### DISCUSSION TITLE
Method to back out from EmailLinkSignInScreen in iOS

### DISCUSSION THREAD

**Discussion by rstieger** [firebase/FirebaseUI-Flutter#80] [Q&A] [answered]
Given an app using SignInScreen from firebase_ui_auth, with EmailLinkAuthProvider as one of the providers:

A user might accidentally click on  the "Sign in with magic link" button. Or they might mistype their email address. In either case, on iOS there is no way to go back to the original SignInScreen, short of terminating and restarting the app. (Android always has a back button available.)

It would be better if EmailLinkSignInScreen/EmailLinkSignInView had the same "Go back" option that other screens have (for example, SMSCodeInputScreen). This could be a boolean parameter to the EmailLinkSignInScreen constructor in case some developers prefer the flow as-is.

**Comment by lesnitsky** [comment, marked as answer]
Implemented in #63 by original author, great job 👍
